Source/WebCore/html/CheckboxInputType.cpp

```diff
@@ -281,8 +286,9 @@ bool CheckboxInputType::matchesIndeterminatePseudoClass() const
 
 void CheckboxInputType::disabledStateChanged()
 {
-    ASSERT(element());
-    if (isSwitch() && element()->isDisabledFormControl()) {
+    RefPtr element = this->element();
+    ASSERT(element);
+    if (isSwitch() && element->isDisabledFormControl()) {
```

Comment:
I think we should split this `if` case into 2 so we don't need to ref the element if `isSwitch()` is false, which is likely common.

Reply:
Done.